src/test/ui/parser/issue-90728.rs

```diff
@@ -0,0 +1,6 @@
+fn main() {
+    a.5.2E+
+    //~^ ERROR: unexpected token: `5.2E+`
+    //~^^ ERROR: expected one of `.`, `;`, `?`, `}`, or an operator, found `5.2E+`
+    //~^^^ ERROR: expected at least one digit in exponent
```

Comment:
I usually write these this way because it lets the `ERROR`s align :)
```suggestion
    //~^ ERROR: unexpected token: `5.2E+`
    //~| ERROR: expected one of `.`, `;`, `?`, `}`, or an operator, found `5.2E+`
    //~| ERROR: expected at least one digit in exponent
```